### ISSUE TITLE
val_loss unavailable to use for early stopping

### ISSUE THREAD

**Issue by asikuna** [asikuna/GALottery-Forecaster#2]
I closed VSC so I can't see the exact error, but I'll update this later including it and try to debug.

**Comment by asikuna** [comment]
The error is `WARNING:tensorflow:Early stopping conditioned on metric val_loss which is not available. Available metrics are: loss` and I circumvented it by making a very large amount for the callback. It's kinda janky but works for the time being. Not sure why the val_loss doesn't appear but maybe it's incompatible with the manually set loss function.

`early_stopping = tf.keras.callbacks.EarlyStopping(monitor='loss', patience=1000000, verbose=1, mode='min')`
